### PR TITLE
Avoid raising exception when epoch is "best"

### PR DESCRIPTION
**Pull request by himkt**

### CODE REVIEW

File: msnc/model.py [hunk 1]
```diff
@@ -145,7 +145,14 @@ def _ischeckpoint(self, epoch):
         return epoch % self.checkpoint_interval == 0
 
     def _save(self, epoch):
-        model_path = self._output_dir_path / '{:04d}.model'.format(epoch)
+        if isinstance(epoch, int):
+            model_name = '{:04d}.model'.format(epoch)
+        elif isinstance(epoch, str):
```
Comment (himkt) [inline msnc/model.py]: It accepts 'self._save('best')` .

**Comment by himkt** [comment]
@jun-harashima :bow:

File: msnc/model.py (hunk 1 above)
Comment (jun-harashima) [reply msnc/model.py]: It is somewhat strange that `epoch` can be a staring. How about giving a file name to the function 🤔 

```
def _save(self, file_name):
   model_path = self._output_dir_path / file_name
   ...
```

Comment (himkt) [reply msnc/model.py]: I totally agree with you, sorry for the dirty workaround. :bow:
I revised my PR, please check it.

**Pull request merged**